### ISSUE TITLE
[FEATURE] specific last item ending in bullet lists

### ISSUE THREAD

**Issue by MathiasWinckel** [RosaeNLG/rosaenlg#148]
**Is your feature request related to a problem? Please describe.**
We often (if not always) have to end the last sentence in a bullet listing differently than the previous ones:
- first sentence ;
- second sentence ;
- last sentence.

Either in reports or contracts generation, we had to do that, that seems a quite common punctuation pattern for bullet listing.
It would definitely be a nice to have feature having this automatically handle, a bit like the separators are done for you.

**Describe the solution you'd like**
Managing automatically a different ending for the last bullet, through a new "_list_last_end_item_" parameter of the assembly:
`eachz elt in ['a', 'b', 'c'] with { mode:'list', list_end_item: ";", list_last_end_item: "."  }`
In case only the _list_end_item_ parameter is given, it would be used as the last ending too.
In case only the _list_last_end_item_ parameter is given, only the last item is ended, with that.
That would be available only for bullet listing.

**Describe alternatives you've considered**
So far, it seemed possible to add conditions in the single _list_end_item_ available parameter to choose from _; or ._ to use. It may be not as hard as writing a last separator during a listing...
But with complex cases, when the bullet list is a mix of partially commented listing and optional sentences, I'm not 100% sure my conditions are correct, and it requires a very clean handling of the conditions all across the items, and may add complexity to maintain that. 
I even considered handling them during a post process step matching the `</li></ul>` with regex instead of doing it with the templates, because it seems maybe easier to maintain since it would rely on far less logic and code writing.

**Additional context**
If you're ok with this feature, I have something ready to suggest: 
basically an _insertEndOfSentences_ method very close to the _insertSeparatorSentences_ in the _AsmManager_, to use when writing the end of sentence only in the "list" case, instead of the current basic if.

**Comment by ludans** [comment]
Thanks @MathiasWinckel .
See PR, it should do the job.
Given the level of detail you provided, I am pretty sure you could do the PR directly by yourself next time.

**Comment by MathiasWinckel** [comment]
Indeed, I could. Actually I already written and tested the code and planned on doing the PR today... 
But thanks for making it.
